### PR TITLE
Add a method to publish a plane using the normal and distance

### PR DESCRIPTION
**Pull request by stephanie-eng**
Add a convenience method to publish a plane using a normal and an offset

### CODE REVIEW

**Comment by stephanie-eng** [comment]
Not sure of the best function name, so if anyone has any better ideas, I'm very open to renaming it.

File: include/rviz_visual_tools/rviz_visual_tools.hpp [hunk 1]
```diff
@@ -517,6 +517,19 @@ class RvizVisualTools
   bool publishABCDPlane(const double A, const double B, const double C, const double D,
                         Colors color = TRANSLUCENT, double x_width = 1.0, double y_width = 1.0);
 
+  /**
+   * \brief Display a plane given a vector normal to the plane and the distance to the plane along
+   *        that normal. The plane normal does not need to given as a unit vector.
```
Comment (AndyZe) [inline include/rviz_visual_tools/rviz_visual_tools.hpp:522]:
```suggestion
   *        that normal. The plane normal does not need to be given as a unit vector.
```

File: include/rviz_visual_tools/rviz_visual_tools.hpp [hunk 2]
```diff
@@ -517,6 +517,19 @@ class RvizVisualTools
...
+   * \param normal - a vector representing the normal of the plane
+   * \param d - the distance to the plane along the vector
+   * \param color - the color of the plane
+   * \param x_width - X-size of the vizualized plane [meters]
+   * \param y_width - Y-size of the visualized plane [meters]
+   * \return true on success
+   */
+  bool publishNormalAndDistancePlane(Eigen::Vector3d normal, double d, Colors color = TRANSLUCENT,
```
Comment (AndyZe) [inline include/rviz_visual_tools/rviz_visual_tools.hpp:530]:
```suggestion
  bool publishNormalAndDistancePlane(const Eigen::Vector3d& normal, const double d, const Colors color = TRANSLUCENT,
```

File: include/rviz_visual_tools/rviz_visual_tools.hpp [hunk 3]
```diff
@@ -517,6 +517,19 @@ class RvizVisualTools
...
+                                     double x_width = 1.0, double y_width = 1.0);
```
Comment (AndyZe) [inline include/rviz_visual_tools/rviz_visual_tools.hpp:531]:
```suggestion
                                     const double x_width = 1.0, const double y_width = 1.0);
```

File: include/rviz_visual_tools/rviz_visual_tools.hpp (hunk 2 above)
Comment (stephanie-eng) [reply include/rviz_visual_tools/rviz_visual_tools.hpp:530]: `d` can't be const as I modify it in the function body, but good point on the other two

**Review by AndyZe — APPROVED** [review]
LGTM. The new variable `D` could be `const`  (Super nitpick.) Feel free to merge

**Pull request merged**